src/commands/service/repl.ts

```diff
@@ -243,45 +243,31 @@ const ensureModuleConfigsForToml = (
   });
 };
 
+const ENTER_PATH_OR_URL_MSG =
+  "Enter path to a service or url to .tar.gz archive";
+
 async function promptForNamePathOrUrl(
   maybeFluenceConfig: FluenceConfig | null,
 ): Promise<string> {
   const serviceNames = Object.keys(maybeFluenceConfig?.services ?? {});
-  const [firstService] = serviceNames;
-
-  if (
-    serviceNames.length === 1 &&
-    firstService !== undefined &&
-    (await confirm({
-      message: `Do you want to select ${color.yellow(
-        firstService,
-      )} service from ${color.yellow(maybeFluenceConfig?.$getPath())}`,
-    }))
-  ) {
-    return firstService;
-  }
 
-  if (
-    serviceNames.length > 1 &&
-    (await confirm({
-      message: `Do you want to select a service from ${color.yellow(
-        maybeFluenceConfig?.$getPath(),
-      )}`,
-    }))
-  ) {
-    return list({
-      message: "Select service",
-      options: serviceNames,
-      oneChoiceMessage() {
-        throw new Error("Unreachable");
-      },
-      onNoChoices() {
-        throw new Error("Unreachable");
-      },
+  const selected = await list({
+    message: "Select service",
+    options:
+      serviceNames.length === 0 ? [] : [...serviceNames, ENTER_PATH_OR_URL_MSG],
```

Comment:
Maybe suggest `ENTER_PATH_OR_URL_MSG` even if there are no services? Didn't get why not

Reply:
I do it down below. If you leave it here it will ask you to select this one weird item in the list and only then you will have to actually enter the path or url

Reply:
got it, thx!